frontend/src/booking/DetailsContainer.js

```diff
@@ -54,15 +80,21 @@ const DetailsContainer = (props) => {
           className="form-value"
         />
         <TextField
+          error={validation("phone")}
+          onBlur={(evt) => handleBlur("phone")(evt)}
           type="text"
-          inputProps={{ maxLength: 40 }}
+          inputProps={{ maxLength: 40, pattern: "[0-9]*" }}
```

Comment:
Would we need to handle phone numbers with the + symbol in front of them? Other than that LGTM

Reply:
I think its fine not handling it. The restaurant reservations are probably only intended to deal with local customers, so the majority of people wouldn't enter the +area code part. Those who do use that format would just change to other format upon finding its not accepting their current number